artifacts/agent/karmada-agent.yaml

```diff
@@ -43,9 +43,10 @@ spec:
             periodSeconds: 15
             timeoutSeconds: 5
           volumeMounts:
-            - name: kubeconfig
+            - name: karmada-kubeconfig
+              subPath: kubeconfig
```

Comment:
What is the purpose of adding a subpath?

Reply:
get it. But I'm concerned about, a container using a Secret as a [subPath](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath) volume mount [does not receive automated Secret updates](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod).